docs/command-reference.md

```diff
@@ -0,0 +1,709 @@
+*   **Global**: `--version`, `-h`/`--help`
+
+### `container registry logout`
+
+Logs out of a registry, removing stored credentials.
+
+**Usage**
+
+```bash
+container registry logout SERVER
+```
+
+Only `--version` and `-h`/`--help` are available.
+
+### `container registry default` commands
+
+The `registry default` group allows setting, unsetting, and inspecting the default registry used when no registry is specified on image references.
+
+*   `container registry default set [OPTIONS] HOST`: Set the default registry.
+    *   `--scheme <scheme>`: registry scheme (`auto`, `http`, `https`)
+    *   **Global**: `--debug`, `--version`, `-h`/`--help`
+*   `container registry default unset (clear)`: Clears the default registry configuration.
+*   `container registry default inspect`: Displays the current default registry, if any.
+
+## System Management
+
+System commands manage the container apiserver, logs, DNS settings and kernel. These are only available on macOS hosts.
+
+### `container system start`
+
+Starts the container services and (optionally) installs a default kernel. It will start the `container-apiserver` and background services.
+
+**Usage**
+
+```bash
+container system start [OPTIONS]
+```
+
+**Options**
+
+*   `-a, --app-root <path>`: application data directory
+*   `--install-root <path>`: path to the installation root directory
+*   `--debug`: enable debug logging for the runtime daemon
+*   `--enable-kernel-install`: install the recommended default kernel
+*   `--disable-kernel-install`: skip installing the default kernel
+  If neither kernel-install flag is provided, you will be prompted to choose whether to install the recommended kernel.
+
+### `container system stop`
+
+Stops the container services and deregisters them from launchd. You can specify a prefix to target services created with a different launchd prefix.
+
+**Usage**
+
+```bash
+container system stop [OPTIONS]
+```
+
+**Options**
+
+*   `-p, --prefix <prefix>`: launchd prefix (default: `com.apple.container.`)
+*   **Global**: `--version`, `-h`/`--help`
+
+### `container system status`
+
+Checks whether the container services are running and prints status information. It will ping the apiserver and report readiness.
+
+**Usage**
+
+```bash
+container system status [OPTIONS]
+```
+
+**Options**
+
+*   `-p, --prefix <prefix>`: launchd prefix to query (default: `com.apple.container.`)
+*   **Global**: `--version`, `-h`/`--help`
+
+### `container system logs`
+
+Displays logs from the container services. You can specify a time interval or follow new logs in real time.
+
+**Usage**
+
+```bash
+container system logs [OPTIONS]
+```
+
+**Options**
+
+*   `--last <duration>`: Fetch logs starting from the specified time period (minus the current time); supported formats: m, h, d (default: 5m)
+*   `-f, --follow`: Follow log output
+*   **Global**: `--debug`, `--version`, `-h`/`--help`
+
+### DNS management (`container system dns`) (macOS only)
```

Comment:
Should we have another subsection here that lists out the different subcommands like we did for the builder management commands? Same for the `container system kernel` section below? Just for consistency